### ISSUE TITLE
Add Tests

### ISSUE THREAD

**Issue by coreyarms** [coreyarms/mysql-types-generator#17]
Should add more/better tests. Maybe mock the `getTableNames` and `getColumnInfo` functions to return a static data set of tables/columns and compare the generated outputs to the expected outputs for various config option combinations?

**Comment by evert** [comment]
Another way to handle this is to create a function that takes a table name + column information (`COLUMN[]`), and returns a `string` with the output. Similarly a function that takes a `COLUMN` and returns the typescript output of a single column

Mocking sucks! But refactoring the important bits into pure functions makes this a lot easier and relatively easy to test.

**Comment by coreyarms** [comment]
Yeah that's a better idea. The only thing that doesn't test is writing the files to disk, which can probably be done separately by testing just the `writeToFile` function.
